packages.dhall

```diff
@@ -1,7 +1,10 @@
 let upstream =
       https://github.com/purescript/package-sets/releases/download/psc-0.13.6-20200507/packages.dhall sha256:9c1e8951e721b79de1de551f31ecb5a339e82bbd43300eb5ccfb1bf8cf7bbd62
 
-let overrides = {=}
+let overrides =
+    { web-dom =
+        upstream.web-dom // { repo = "https://github.com/srghma/purescript-web-dom.git", version = "patch-1" }
```

Comment:
This is OK while this is a draft PR, but I think Halogen versions should not have any overrides or additions except for libraries that are used in test code.

Let's make sure that `purescript-web-dom` gets updated before this PR merges.